pkgs/by-name/do/dosbox-staging/package.nix

```diff
@@ -1,14 +1,14 @@
 {
   lib,
```

Comment:
Why `lib` is breaking `ASCIIbetical` order?

Reply:
Historical and aesthetic reasons.

`lib` was (almost) always first since it was split from stdenv.
And since it is almost omnipresent an rarely subjected to overriding, it makes some sense to put it first.

Reply:
My bias is for alphabetical ordering, but I can understand the need of `ASCIIbetical`. Since it is the alphabetical ordering that computers are usually doing.

A new person, with no historical context, can understand alphabetical ordering. But having to explain some historical context for the person getting `lib` right (whatever that is) is extra work. I'd rather not have to.

Reply:
`lib` is mostly off anyway. It is, roughly speaking just a collection of "macros".
It is not used as a `buildInput` or anything that affects the build.

This is mostly aesthetic at this point.